processor/attributesprocessor/README.md

```diff
@@ -46,8 +46,10 @@ For the actions `insert`, `update` and `upsert`,
 - key: <key>
   action: {insert, update, upsert}
   # FromContext specifies the context value to use to populate the attribute value. 
-  # The context values would be searched in receiver's transport protocol additional information
-  # like GRPC Metadata or HTTP Headers. 
+  # If key is prefixed with `metadata.` or has no prefix then values are searched 
```

Comment:
```suggestion
  # If the key is prefixed with `metadata.`, the values are searched 
```

If we ever intend to remove the support for unqualified keys, it's better to not mention the "no prefix" here.